Chummer/data/gear.xml

```diff
@@ -1532,6 +1532,68 @@
       </weaponbonus>
       <ammoforweapontype>firefighting cannons</ammoforweapontype>
     </gear>
+	<gear>
+      <id>d71ea9c1-9b5f-428e-b849-7d665eb31c5e</id>
+      <name>Ammo: Sling-Shot Acid Capsules</name>
+      <category>Ammunition</category>
+      <rating>0</rating>
+      <source>SOTG</source>
+      <page>13</page>
+      <avail>-</avail>
+      <cost>100</cost>
+      <costfor>10</costfor>
+      <weaponbonus>
+        <damagereplace>6P Acid</damagereplace>
```

Comment:
"Acid" should be in its own `<damagetype>` node. That way the 6P will properly be recognized as 6 damage of Physical type, allowing things like damage-modifying weapon accessories to work.

Reply:
Fixed.